java/pom.xml

```diff
@@ -52,14 +52,17 @@
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-release-plugin</artifactId>
-                <version>2.2.1</version>
+                <artifactId>maven-jar-plugin</artifactId>
```

Comment:
mvn package already does that - and we don't need hebmorph-lucene.jar - we need the file name to be versioned